website/client/src/components/achievements/newStuff.vue

```diff
@@ -36,33 +51,110 @@
 </template>
 
 <style lang='scss'>
-  @import '~@/assets/scss/static.scss';
-  #new-stuff {
-    .modal-body .modal-body {
-      padding-top: 0rem;
-    }
+@import '~@/assets/scss/static.scss';
+</style>
+
+<style lang='scss' scoped>
+.modal-body {
+  padding-top: 2em;
+}
+.bailey {
+  margin-bottom: 60px;
+  .title {
+    display: inline;
+  }
+
+  .draft {
+    margin-right: 10px;
+  }
+
+  h2 {
+    color: #4F2A93;
+  }
+
+  div {
+    font-size: 16px;
+  }
+
+  .center-block {
+    display: block;
+    margin: auto;
+  }
+}
+.bailey-last {
+  margin-bottom: 10px;
+}
+</style>
+
+<style>
+  .bailey .markdown-img {
+    display: block;
+    margin: auto;
+  }
+
+  .bailey .markdown h1 {
+    color: #4F2A93 !important;
+    font-size: 2.5rem !important;
+    margin-top: 4rem !important;
+    margin-bottom: 1rem !important;
+  }
+
+  .bailey .markdown h2 {
+    color: #4F2A93 !important;
+    font-size: 2rem !important;
+    margin-top: 3.5rem !important;
+    margin-bottom: 1rem !important;
+  }
+
+  .bailey .markdown h3 {
+    color: #4F2A93 !important;
+    font-size: 1.75rem !important;
+    margin-top: 3rem !important;
+    margin-bottom: 1rem !important;
+  }
+
+  .bailey .markdown h4 {
+    color: #4F2A93 !important;
+    font-size: 1.5rem !important;
+    margin-top: 2rem !important;
+    margin-bottom: 1rem !important;
+  }
+
+  .bailey div {
+    font-size: 16px;
+  }
+
+  .bailey .center-block {
+    display: block;
+    margin: auto;
   }
 </style>
 
 <script>
-import axios from 'axios';
+import moment from 'moment';
+import habiticaMarkdown from 'habitica-markdown';
 import { mapState } from '@/libs/store';
 
 export default {
+  filters: {
+    date (value) {
+      // @TODO: Vue doesn't support this so we cant user preference
+      return moment(value).format('l');
+    },
+  },
   data () {
     return {
-      html: '',
+      posts: [],
     };
   },
   computed: {
     ...mapState({ user: 'user.data' }),
   },
   async mounted () {
-    this.$root.$on('bv::show::modal', async modalId => {
-      if (modalId !== 'new-stuff') return;
-      const response = await axios.get('/api/v4/news');
-      this.html = response.data.html;
-    });
+    this.posts = (await this.$store.dispatch('user:getNews')).data.data;
```

Comment:
I've noticed that this API call on delta is made as soon as the page loads even if the bailey modal isn't shown, probably because it gets `mounted` immediately. it should probably be made only when the modal is shown by listening to `this.$root.$emit('bv::show::modal', 'new-stuff');`